webapp-django/crashstats/crashstats/models.py

```diff
@@ -900,7 +899,6 @@ class SignaturesByBugs(SocorroMiddleware):
 
 
 class SignatureFirstDate(SocorroMiddleware):
```

Comment:
This is a Django-model using middleware. I think I'm going to go with this method of doing things for now. It supports the API documentation and other things, so it's convenient.